--- src/db/schema.rs.orig	2022-09-21 11:58:12.604181040 +0200
+++ src/db/schema.rs	2022-09-21 11:59:53.941227238 +0200
@@ -2,19 +2,19 @@
 
 diesel::table! {
     blocked_guilds (guild) {
-        guild -> Integer,
+        guild -> BigInt,
     }
 }
 
 diesel::table! {
     blocked_users (user) {
-        user -> Integer,
+        user -> BigInt,
     }
 }
 
 diesel::table! {
     settings (guild) {
-        guild -> Integer,
+        guild -> BigInt,
         game -> Nullable<Integer>,
         prefix -> Nullable<Text>,
     }
@@ -23,9 +23,9 @@
 diesel::table! {
     subscriptions (game, channel, tags) {
         game -> Integer,
-        channel -> Integer,
+        channel -> BigInt,
         tags -> Text,
-        guild -> Nullable<Integer>,
+        guild -> Nullable<BigInt>,
         events -> Integer,
     }
 }
@@ -33,8 +33,8 @@
 diesel::table! {
     subscriptions_exclude_mods (game, channel, mod_id) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Nullable<Integer>,
+        channel -> BigInt,
+        guild -> Nullable<BigInt>,
         mod_id -> Integer,
     }
 }
@@ -42,8 +42,8 @@
 diesel::table! {
     subscriptions_exclude_users (game, channel, user) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Nullable<Integer>,
+        channel -> BigInt,
+        guild -> Nullable<BigInt>,
         user -> Text,
     }
 }
