diff --git a/src/db/schema.rs b/src/db/schema.rs
index f1eb2dc..0541c98 100644
--- a/src/db/schema.rs
+++ b/src/db/schema.rs
@@ -2,19 +2,19 @@
 
 diesel::table! {
     blocked_guilds (guild) {
-        guild -> Integer,
+        guild -> BigInt,
     }
 }
 
 diesel::table! {
     blocked_users (user) {
-        user -> Integer,
+        user -> BigInt,
     }
 }
 
 diesel::table! {
     settings (guild) {
-        guild -> Integer,
+        guild -> BigInt,
         game -> Nullable<Integer>,
     }
 }
@@ -22,9 +22,9 @@ diesel::table! {
 diesel::table! {
     subscriptions (game, channel, tags) {
         game -> Integer,
-        channel -> Integer,
+        channel -> BigInt,
         tags -> Text,
-        guild -> Integer,
+        guild -> BigInt,
         events -> Integer,
     }
 }
@@ -32,8 +32,8 @@ diesel::table! {
 diesel::table! {
     subscriptions_exclude_mods (game, channel, mod_id) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Integer,
+        channel -> BigInt,
+        guild -> BigInt,
         mod_id -> Integer,
     }
 }
@@ -41,8 +41,8 @@ diesel::table! {
 diesel::table! {
     subscriptions_exclude_users (game, channel, user) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Integer,
+        channel -> BigInt,
+        guild -> BigInt,
         user -> Text,
     }
 }
