--- src/db/schema.rs.orig	2019-05-25 23:19:03.711325654 +0200
+++ src/db/schema.rs	2019-05-25 23:22:26.525025890 +0200
@@ -1,12 +1,12 @@
 table! {
     blocked_guilds (guild) {
-        guild -> Integer,
+        guild -> BigInt,
     }
 }
 
 table! {
     blocked_users (user) {
-        user -> Integer,
+        user -> BigInt,
     }
 }
 
--- src/db/schema.rs.orig	2019-02-10 21:15:15.827715165 +0100
+++ src/db/schema.rs	2019-02-10 21:15:36.054407291 +0100
@@ -1,6 +1,6 @@
 table! {
     settings (guild) {
-        guild -> Integer,
+        guild -> BigInt,
         game -> Nullable<Integer>,
         prefix -> Nullable<Text>,
     }
--- src/db/schema.rs.orig	2019-02-24 22:39:33.641530261 +0100
+++ src/db/schema.rs	2019-02-24 22:40:13.184879005 +0100
@@ -9,8 +9,8 @@
 table! {
     subscriptions (game, channel, tags) {
         game -> Integer,
-        channel -> Integer,
+        channel -> BigInt,
         tags -> Text,
-        guild -> Nullable<Integer>,
+        guild -> Nullable<BigInt>,
         events -> Integer,
     }
 }
--- src/db/schema.rs.orig
+++ src/db/schema.rs
@@ -30,8 +30,8 @@ table! {
 table! {
     subscriptions_exclude_mods (game, channel, mod_id) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Nullable<Integer>,
+        channel -> BigInt,
+        guild -> Nullable<BigInt>,
         mod_id -> Integer,
     }
 }
diff --git a/src/db/schema.rs b/src/db/schema.rs
index 3d8b460..a28e031 100644
--- a/src/db/schema.rs
+++ b/src/db/schema.rs
@@ -40,8 +40,8 @@ table! {
 table! {
     subscriptions_exclude_users (game, channel, user) {
         game -> Integer,
-        channel -> Integer,
-        guild -> Nullable<Integer>,
+        channel -> BigInt,
+        guild -> Nullable<BigInt>,
         user -> Text,
     }
 }
